releasenotes/notes/move-correlation-ids-a5a144e900decbc5.yaml

```diff
@@ -0,0 +1,5 @@
+---
+other:
+  - |
+    `ddtrace.helpers.get_correlation_ids` was moved to be an instance method in `ddtrace.Tracer`. It no longer
+    looks at the global tracer instance by default.
```

Comment:
```suggestion
    `ddtrace.helpers.get_correlation_ids` was moved to be an instance method in `ddtrace.Tracer`.
```

I think this is implied by the fact that it is an instance method. What do you think?